homeassistant/components/plex/server.py

```diff
@@ -121,8 +130,22 @@ def update_platforms(self):
         _LOGGER.debug("Updating devices")
 
         available_clients = {}
+        ignored_clients = set()
         new_clients = set()
 
+        monitored_users = self.accounts
+        known_accounts = set(self.options[MP_DOMAIN].get(CONF_MONITORED_USERS, []))
+        if known_accounts:
+            monitored_users = {
+                user
+                for user in self.options[MP_DOMAIN][CONF_MONITORED_USERS]
+                if self.options[MP_DOMAIN][CONF_MONITORED_USERS][user]["enabled"]
```

Comment:
Can it be that a new user is not represented in `self.options` yet ?

Reply:
Yes, that's what the next block of code handles. Or do you mean something else?